compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -34,7 +34,22 @@ macro_rules! err_ub {
 
 #[macro_export]
 macro_rules! err_ub_format {
-    ($($tt:tt)*) => { err_ub!(Ub(format!($($tt)*))) };
+    ($msg:expr $(, $($name:ident = $value:expr),* $(,)?)?) => {{
```

Comment:
What are those `=` productions about? Looks like the macro now needs a doc comment to explain how to use it.